services-js/registry-certs/client/birth/questions/YesNoUnsureComponent.tsx

```diff
@@ -20,47 +19,36 @@ interface Props {
  */
 export default function YesNoUnsureComponent(props: Props): JSX.Element {
   function radioItem(
-    value: YesNoUnknownAnswer,
-    text: string
+    itemValue: YesNoUnknownAnswer,
+    itemText: string
   ): React.ReactChild {
+    // map values to the icons to use
+    const iconName =
```

Comment:
One of the things I miss about CoffeeScript is how all of its statements had values, so you could easily use a `switch` in this scenario. Doing one level of ternary operator nesting is fine though, if it means you can keep the `const`.